internal/controlplane/handlers_authz.go

```diff
@@ -530,32 +495,70 @@ func (s *Server) removeInvite(
 	role authz.Role,
 	email string,
 ) (*minder.RemoveRoleResponse, error) {
-	prj := targetPrj.String()
-	// Get all invitations for this email, project and role
-	inviteToRemove, err := s.store.GetInvitationByEmailAndProjectAndRole(ctx, db.GetInvitationByEmailAndProjectAndRoleParams{
+	// Get all invitations for this email and project
+	invitesToRemove, err := s.store.GetInvitationsByEmailAndProject(ctx, db.GetInvitationsByEmailAndProjectParams{
 		Email:   email,
 		Project: targetPrj,
-		Role:    role.String(),
```

Comment:
oh this removed line seems like you were doing that before?

Reply:
Exactly, my reasoning was we discussed whether a user can have more than 1 role within a project (not allowed for now), so I decided to allow for this in the future.